<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="a">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="operacja">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>^</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="b">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="oblicz">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wynik">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Autor">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>???</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutorCheckBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pokaż autora</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="Data">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>304</width>
      <height>163</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DataLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybrana data</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="NrTelBar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="NrTelLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybrany numer</string>
    </property>
   </widget>
   <widget class="QLabel" name="NotesNazwisko">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj nazwisko:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NotesLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="NotesButton">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>120</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QLabel" name="NotesLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Notes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(255, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Sum1Line">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Sum1Label1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj liczbę: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sum1Button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sum1Label2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podane liczby:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Sum1List">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>151</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Sum1Label3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Liczba pozycji: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Sum1Count">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sum1Button2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyzeruj</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sum1Label4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Suma: </string>
    </property>
   </widget>
   <widget class="QLabel" name="Sum1Sum">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sum2Label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Podaj liczbę:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Sum2Box">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Sum2Button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>++</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NotesButtonWrite">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NotesButtonRead">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Odczyt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NotesButtonClear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyczyść</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="WindowColorBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Kolor formularza</string>
    </property>
    <widget class="QRadioButton" name="WindowColorButtonRed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>czerwony</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="WindowColorButtonGreen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>zielony</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="WindowColorButtonGray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>szary</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="KalkLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kalkulator</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKalkulator">
    <property name="title">
     <string>Kalkulator</string>
    </property>
    <addaction name="actionPokaz"/>
    <addaction name="actionUkryj"/>
   </widget>
   <widget class="QMenu" name="menuEdycja_notesu">
    <property name="title">
     <string>Edycja notesu</string>
    </property>
    <addaction name="actionKopiuj"/>
    <addaction name="actionWytnij"/>
    <addaction name="actionWklej"/>
   </widget>
   <widget class="QMenu" name="menuKomunikat">
    <property name="title">
     <string>Komunikat</string>
    </property>
    <addaction name="actionk1"/>
    <addaction name="actionk2"/>
    <addaction name="actionAutor"/>
    <addaction name="actionWy_czanie"/>
    <addaction name="actionTestK"/>
   </widget>
   <addaction name="menuKalkulator"/>
   <addaction name="menuEdycja_notesu"/>
   <addaction name="menuKomunikat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPokaz">
   <property name="text">
    <string>Pokaż</string>
   </property>
   <property name="toolTip">
    <string>Pokaż</string>
   </property>
  </action>
  <action name="actionUkryj">
   <property name="text">
    <string>Ukryj</string>
   </property>
  </action>
  <action name="actionKopiuj">
   <property name="text">
    <string>Kopiuj</string>
   </property>
  </action>
  <action name="actionWytnij">
   <property name="text">
    <string>Wytnij</string>
   </property>
  </action>
  <action name="actionWklej">
   <property name="text">
    <string>Wklej</string>
   </property>
  </action>
  <action name="actionk1">
   <property name="text">
    <string>k1</string>
   </property>
  </action>
  <action name="actionk2">
   <property name="text">
    <string>k2</string>
   </property>
  </action>
  <action name="actionAutor">
   <property name="text">
    <string>Autor</string>
   </property>
  </action>
  <action name="actionWy_czanie">
   <property name="text">
    <string>Wyłączanie</string>
   </property>
  </action>
  <action name="actionTestK">
   <property name="text">
    <string>TestK</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
